components/world-4/breeding/pets.tsx

```diff
@@ -0,0 +1,522 @@
+                        label="Breedabilli Equipped"
+                        onChange={(event) => {
+                            setStarSignEquipped(event.target.checked);
+                            if(!event.target.checked) {
+                                setSilkrode(false);
+                            }
+                        }}
+                        disabled={starSignInfinity}
+                    />
+                    <TipDisplay
+                        heading="Breedabilli"
+                        size='medium'
+                        maxWidth='medium'
+                        body={
+                            <Box>
+                                <Text size='small'>Looks like you unlocked the Breedabilli star sign</Text>
+                                {
+                                    starSignInfinity ?
+                                    <Text margin={{top:'xsmall'}} size='small'>You always get the star sign bonus thanks to the Infinite Stars Rift bonus</Text>
+                                    :
+                                    <Text margin={{top:'xsmall'}} size='small'>To avoid character checking for a global page, use this checkbox to consider it equipped or not</Text>
+                                }
+                            </Box>
+                        }
+                    >
+                        <CircleInformation size="small" />
+                    </TipDisplay>
+                </Box>
+            }
+            {
+                starSignUnlocked && (activeTab == "Shiny" || activeTab == "Breedability") &&
+                <Box direction='row' gap='xsmall'>
+                    <CheckBox
+                        checked={silkRodeChip}
+                        label="Silkrode Nanochip Equipped"
+                        onChange={(event) => setSilkrode(event.target.checked)}
+                        disabled={!starSignEquipped}
+                    />
+                    <TipDisplay
+                        heading="Silkrode Nanochip"
+                        size='medium'
+                        maxWidth='medium'
+                        body={
+                            <Box>
+                                <Text size='small'>You can check this checkbox to get accurate values when equipping the Silkrode Nanochip in the Lab (double star sign bonus)</Text>
+                            </Box>
+                        }
+                    >
+                        <CircleInformation size="small" />
+                    </TipDisplay>
+                </Box>
+            }
+            </Box>
+            {activeTab == "All" && <AllPetDisplay />}
+            {activeTab == "Shiny" && <ShinyDisplay silkRodeChip={silkRodeChip} starSignEquipped={starSignEquipped} />}
+            {activeTab == "Breedability" && <BreedabilityDisplay silkRodeChip={silkRodeChip} starSignEquipped={starSignEquipped} />}
+        </Box>
+    )
+}
+
+function AllPetDisplay() {
```

Comment:
Can you tell me your thought process behind this tab? right now I'm not sure how I would use it so I'm trying to understand what your vision is for it.

Reply:
It's a way to show all available pets, and their power and let you know which ones you unlocked and which ones you didn't (if not they'll be greyed out)
If you haven't unlocked all pets yet the only way to know what is coming (like which one to focus on for their power) is to go to the wiki as it's hidden in-game

If you think it's not really a good idea it's easy to remove it

Reply:
By hovering the power icon you get the description of it, and by hovering the pet itself you get the gene level and the next level cost
You also get to know how many genes you get for each color

Reply:
I think let's keep it as is for now, we can potentially enhance this later to maybe not show gene info once you unlocked all pets / unlocked a certain pet.

I haven't played in a while but I don't recall a reason to care about genes once you are done with unlocking all pets 🤔